app/src/main/java/org/simple/clinic/editpatient/deletepatient/DeletePatientScreen.kt

```diff
@@ -156,7 +156,7 @@ class DeletePatientScreen(context: Context, attrs: AttributeSet) : ConstraintLay
   }
 
   override fun showHomeScreen() {
-    screenRouter.clearHistoryAndPush(HomeScreenKey, RouterDirection.BACKWARD)
```

Comment:
In this case, going backward is incorrect since we are replacing the entire history with a new one. We should have used `Replace` here from the start.

Reply:
In a future commit, we used `Forward`. Just for my understanding, are these being used to determine the screen transitions or? 🤔

Reply:
Yeah, they will be used to decide the screen transitions only.